README.md

```diff
@@ -1 +1,158 @@
-# Portfolio-part-2-Microverse
\ No newline at end of file
+# Portfolio
+Portfolio: setup and mobile first
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+  - [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+
+**[Portfolio]** It is a showcase to show my current and future projects.
+
+## 🛠 Built With <a name="built-with-">'Jonathan-Arias'</a>
+
+### Tech Stack <a name="tech-stack">HTML, CSS</a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+  </ul>
+<ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+- **[[Descriptive_README_File]**
+- **[Mobile_first_project]**
+- **[Linterns_Best_Practices]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+1. Download the repo or clone it
+
+2. Use Live server extention 
+ID: ritwickdey.LiveServer
+Or run it via GitHub Pages
+### Prerequisites
+
+In order to run this project you need:
+
+Visual Studio or any other editor such as Sublime Text
+A localhost or a github pages set up
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+
+### Install
+
+Install this project with:
+
+No installation needed
+
+### Usage
+
+To run the project, execute the following command:
+
+Open a live server or a localhost to test it local
+Or
+Uploaded to Github and use Github Pages
+
+
+### Run tests
+
+To run tests, run the following command:
+
+No tests needed at the moment
+
+### Deployment
+
+You can deploy this project using:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jonathan Arias Gallego**
+
+- GitHub: [sinruidotec](https://github.com/sinruidotec)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/jonathan-undefined-385b3926a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Section_2]**
```

Comment:
- [ ] [Optional]  👋 Nice work! I noticed that you have included a feature called "Section 2" under the Future Features section. However, the content of this section is not meaningful or descriptive, which can be confusing for visitors.

🚀 When mentioning future features, it's important to provide specific and relevant details to give readers a clear idea of what to expect. Instead of "Section 2," please consider providing a meaningful description of the intended future feature.

For example:
- Add User Profile Page.
- Add payment gateways.

**hints: you can further improve them by describing the points in one or two lines.**